### ISSUE TITLE
Sound threads

### ISSUE THREAD

**Issue by Soundthreadsded** [dnsev/4cs#69]
Recently I've been wanting to bring sound threads back and a while back I heard about a way to make new sound images but I never got around to it. What is this method and does it still work?

**Comment by dnsev** [comment]
https://dnsev.github.io/4cs/#about/codecs/embed.exe

This method may still work, though it only allows .pngs (with lower visual quality.)
Try it if you want, though I doubt it will catch on due to the quality and decoding time constraints.

**Comment by Soundthreadsded** [comment]
I just tried that way and it doesnt work. RIP sound threads.

**Comment by coty-crg** [comment]
http://crossedshadows.com/4chanMusic.zip

Knock yourself out.

**Comment by Soundthreadsded** [comment]
Have you tried it recently? Does it work?

**Comment by coty-crg** [comment]
Yeah. Here, I uploaded a handful of pictures I made with it. They all work on 4chan. http://crossedshadows.com/soundthread_oc.zip

**Comment by Soundthreadsded** [comment]
I just tried posting one and it didnt work.

**Comment by coty-crg** [comment]
I dumped literally all of them on /v/ last night as a sound thread; they all worked. I just now posted http://boards.4chan.org/v/thread/255462332#p255468607 (just some random thread) and it worked. 

Maybe it only works on specific boards? I haven't done a lot of testing.

**Comment by Soundthreadsded** [comment]
I realized shortly after that the thread was deleted when I tried to post
it. I'll try again. I posted it on /v/ so if you're right it should work.

**Comment by Soundthreadsded** [comment]
Nope it didnt work again.

**Comment by coty-crg** [comment]
I'm using chrome; I don't know if that makes a difference. Could be a player bug.

**Comment by Soundthreadsded** [comment]
I was using Firefox so if I have to I'll try using chrome.
